src/frontend/citizen-portal/src/app/components/dispute-ticket-stepper/dispute-ticket-stepper.component.ts

```diff
@@ -235,37 +260,28 @@ export class DisputeTicketStepperComponent implements OnInit {
    */
   private submitDispute(): void {
     const data: DialogOptions = {
-      titleKey: 'Submit request',
+      titleKey: "Submit request",
       messageKey:
-        'When your request is submitted for adjudication, it can no longer be updated. Are you ready to submit your request?',
-      actionTextKey: 'Submit request',
-      cancelTextKey: 'Cancel',
+        "When your request is submitted for adjudication, it can no longer be updated. Are you ready to submit your request?",
+      actionTextKey: "Submit request",
+      cancelTextKey: "Cancel",
       icon: null,
     };
     this.dialog.open(ConfirmDialogComponent, { data }).afterClosed()
       .subscribe((response: boolean) => {
         if (response) {
-          // const payload = this.disputeFormStateService.jsonTicketDispute;
-          // payload.violationTicketNumber = this.ticket.violationTicketNumber;
-          // payload.violationTime = this.ticket.violationTime;
-
-          // this.busy = this.disputeResource
-          //   .createTicketDispute(payload)
-          //   .subscribe((newDisputeTicket: TicketDisputeView) => {
-          //     // newDisputeTicket.additional = {
-          //     //   _isCourtRequired:true,
-          //     //   _isReductionRequired: true,
-          //     //   _isReductionNotInCourt: true
-          //     // }
-          //     newDisputeTicket.countList = this.countDataList;
-          //     this.disputeService.ticket$.next(newDisputeTicket);
-
-          //     this.router.navigate([
-          //       AppRoutes.disputePath(AppRoutes.SUBMIT_SUCCESS),
-          //     ], {
-          //       queryParams: this.countDataList,
-          //     });
-          //   });
+          this.busy = this.disputesService.apiDisputesCreatePost(<NoticeOfDispute>{
+            ...this.ticket,
+            ...this.form.value,
+            legal_representation: this.legalRepresentationForm.value
+          }).subscribe(res => {
+            let test;
+            // this.router.navigate([
```

Comment:
lines 279 to 283 should be part of a pr for 1251 -- will you have a separate PR for that?